kafka-connect-neo4j/src/main/kotlin/streams/kafka/connect/sink/Neo4jService.kt

```diff
@@ -144,4 +148,19 @@ class Neo4jService(private val config: Neo4jSinkConnectorConfig):
             throw ConnectException(exceptionMessages)
         }
     }
+
+    fun writeDataSync(data: Map<String, List<List<StreamsSinkEntity>>>) {
```

Comment:
Your PR adds this function but I can't see that it's called anywhere.  I think this ties to the Tesla ticket and it's a good addition I want to keep, maybe I just missed how this gets referenced?

Need a config option that ties to the use of this rather than writeData

Reply:
yep that was accidental, will be in another PR